demo/README.md

```diff
@@ -1,134 +1,45 @@
 # ArangoDB Spark Datasource Demo
 
```

Comment:
Perhaps it might be helpful to list or give an overview of what each of the demos is doing (Read, Write, ReadWrite)?